scipy/optimize/tests/test_minpack.py

```diff
@@ -394,6 +394,13 @@ def test_concurrent_with_gradient(self):
 
 
 class TestCurveFit(object):
+    def test_empty_array_with_bounds(self):
+            # This test is regarding the issue #9864
```

Comment:
indentation is wrong here, needs to be 4 spaces

Reply:
I corrected the indentation.